projects.json

```diff
@@ -2912,5 +2912,59 @@
         "tags": "sourcekit"
       }
     ]
+  },
+  {
+    "repository": "Git",
+    "url": "https://github.com/pointfreeco/swift-nonempty.git",
+    "path": "NonEmpty",
```

Comment:
Please add this configuration between the `NetService` and `ObjectMapper` configurations. We try to keep them sorted by `path`.